modules/checklist/steps/step-base.php

```diff
@@ -22,6 +23,10 @@ abstract class Step_Base {
 	const MARKED_AS_COMPLETED_KEY = 'is_marked_completed';
 	const IMMUTABLE_COMPLETION_KEY = 'is_completed';
 
+	const SITE_SETTINGS_TAB = 5;
+
+	const SITE_IDENTITY_TAB = 'settings-site-identity';
```

Comment:
I don't think this belongs in the base. It is specific to one step only.

Reply:
My fault, I thought there are more steps that will use it